### PR TITLE
Feature header section

### PR DESCRIPTION
**Pull request by keptup**

### CODE REVIEW

File: src/components/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+header {
+    background-color: black;
+    height: 80px;
+    display: flex;
+    align-items: center;
+}
+
+header .vataipvat-logo {
```
Comment (spukas) [inline src/components/Header/Header.css]: Do not need to specify `header` tag here, classname should be enough.

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -1,11 +1,63 @@
...
+          </a>
+          <a
+            href="https://www.youtube.com/channel/UCIIFHn_BKIq765WC5_ggHHg"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img
+              src={youtubeLogo}
+              alt="youtube-log"
+              className="social-links__logo youtube-log"
```
Comment (spukas) [inline src/components/Header/Header.js]: Typing error `youtube-log`

File: src/components/Header/Header.css (hunk 1 above)
Comment (keptup) [reply src/components/Header/Header.css]: Fixed

File: src/components/Header/Header.js (hunk 2 above)
Comment (keptup) [reply src/components/Header/Header.js]: Fixed

**Pull request merged**